website/content/api-docs/system/plugins-catalog.mdx

```diff
@@ -137,14 +147,21 @@ supplied name.
 - `type` `(string: <required>)` – Specifies the type of this plugin. May be
   "auth", "database", or "secret".
 
-- `version` `(string: "")` - Specifies the semantic version of this plugin.
+- `oci_image` `(string: "")` - Specifies OCI image to run. If specified, setting `command`,
+  `args`, and `env` will update the container's entrypoint, args, and environment
+  variables (append-only) respectively.
+
+- `runtime` `(string: "")` - Specifies Vault plugin runtime to use if `oci_image` is specified.
+
+- `version` `(string: "")` - Specifies the semantic version of the plugin. Used as the tag
+  when specifying `oci_image`, but with any leading 'v' trimmed.
 
 - `sha256` `(string: <required>)` – This is the SHA256 sum of the plugin's
-  binary. Before a plugin is run it's SHA will be checked against this value, if
-  they do not match the plugin can not be run.
+  binary or the OCI image. Before a plugin is run, its SHA will be checked against this value.
+  If they do not match the plugin can not be run.
 
-- `command` `(string: <required>)` – Specifies the command used to execute the
-  plugin. This is relative to the plugin directory. e.g. `"myplugin"`.
+- `command` `(string: "")` - Specifies the command to spawn the plugin. This defaults
+  to the name of the plugin if both `oci_image` and `command` are unspecified.
```

Comment:
This is true on the CLI, but the API requires it to be specified (the CLI is the bit that does the defaulting).

```suggestion
- `command` `(string: <required>)` - Specifies the command used to execute the
  plugin. This is relative to the plugin directory. e.g. `"myplugin"`, or if `oci_image`
  is also specified, it is relative to the image's working directory.
```